docs-chef-io/content/inspec/resources/docker_image.md

```diff
@@ -11,142 +11,145 @@ platform = "linux"
     }
+  }
+
+Additionally, all keys of the low-level information are valid properties and can be passed in three ways when writing the test.
 
-#### Additionally, all keys of the low-level information are valid properties and can be passed in three ways when writing the test.
 - Serverspec's syntax
-```
+
   its(['key']) { should eq some_value }
   its(['key1.key2.key3']) { should include some_value }
-```
+
 - InSpec's syntax
-```
+
   its(['key']) { should eq some_value }
   its(['key1', 'key2', 'key3']) { should include some_value }
-```
+
 - Combination of Serverspec and InSpec
-```
+
   its(['key1.key2', 'key3']) { should include some_value }
-```
 
 ## Matchers
 
 For a full list of available matchers, please visit our [matchers page](/inspec/matchers/).
 
 ### exist
 
-The `exist` matcher tests if the image is available on the node:
+The `exist` matcher tests if the image is available on the node.
 
-    it { should exist }
+  it { should exist }
 
 ## Examples
-### Test if a docker image exists and verify the image properties: `id`, `image`, `repo` and `tag`.
-
-    describe docker_image('alpine:latest') do
-      it { should exist }
-      its('id') { should eq 'sha256:4a415e...a526' }
-      its('image') { should eq 'alpine:latest' }
-      its('repo') { should eq 'alpine' }
-      its('tag') { should eq 'latest' }
-    end
 
-### Test if a docker image exists and verify the low level information: `Architecture`, `Config.Cmd`, `Os`, and `GraphDriver`
-
-    describe docker_image('ubuntu:latest') do
-      it { should exist }
-      its(['Architecture']) { should eq 'arm64' }
-      its(['Config.Cmd']) { should include 'bash' }
-      its(['GraphDriver.Data.MergedDir']) { should include "/var/lib/docker/overlay2/4336ba2a87c8d82abaa9ee5afd3ac20ea275bf05502d74d8d8396f8f51a4736c/merged" }
-      its(:inspection) { should include 'Architecture' => 'arm64' }
-      its(:inspection) { should_not include 'Architecture' => 'i386' }
-      its(:inspection) { should include "GraphDriver" =>
-        {
-          "Data" => {
-            "MergedDir" => "/var/lib/docker/overlay2/4336ba2a87c8d82abaa9ee5afd3ac20ea275bf05502d74d8d8396f8f51a4736c/merged",
-            "UpperDir" => "/var/lib/docker/overlay2/4336ba2a87c8d82abaa9ee5afd3ac20ea275bf05502d74d8d8396f8f51a4736c/diff",
-            "WorkDir"=> "/var/lib/docker/overlay2/4336ba2a87c8d82abaa9ee5afd3ac20ea275bf05502d74d8d8396f8f51a4736c/work"
-          },
-          "Name" => "overlay2"
-        }
+### Test if a docker image exists and verifies the image properties: ID, image, repository, and tag
+
+  describe docker_image('ALPINE:LATEST') do
+    it { should exist }
+    its('id') { should eq 'sha256:4a415e...a526' }
+    its('image') { should eq 'ALPINE:LATEST' }
+    its('repo') { should eq 'ALPINE' }
+    its('tag') { should eq 'LATEST' }
+  end
+
+### Test if a docker image exists and verifies the low-level information: Architecture, Config.Cmd, OS, and GraphDriver
```

Comment:
OS can be removed. This was something mentioned from my end initially but no test for Os is specified in the example.